### DISCUSSION TITLE
Why not use 'isServer' ?

### DISCUSSION THREAD

**Discussion by stakbucks** [TanStack/query#7532] [General] [answered]
I find a lot of code `typeof window !== undefined`, when there is `isServer` from the core utils package.
Is there a reason for this?

**Comment by TkDodo** [comment]
you gotta be a lot more specific please. Where do we use `typeof window !== undefined` ?

**Reply by stakbucks** [reply]
Sorry for my bad question 😓 
I was curious about  [this case](https://github.com/TanStack/query/blob/main/packages/react-query-next-experimental/src/HydrationStreamProvider.tsx#L101)

**Reply by TkDodo** [reply, marked as answer]
sure, we can use `isServer` here. This was contributed as an experimental package and we didn't really go through nits like this :) Feel free to file a PR

**Reply by stakbucks** [reply]
Thanks :)
